packages/material-ui/src/MenuItem/MenuItem.js

```diff
@@ -7,24 +7,27 @@ import ListItem from '../ListItem';
 export const styles = theme => ({
   /* Styles applied to the root element. */
   root: {
-    ...theme.typography.subtitle1,
+    ...theme.typography.body1,
```

Comment:
This needs special changelog attention. It's likely this is considered a breaking change (using  theme override to change MenuItem typography).

Reply:
The specification seems to encourage body2 but it contradicts all the implementatin of Google on mobile I can look at. I think that body1 is fine here. The intent is to use the default typography people use on the rest of the page.

Reply:
> I think that body1 is fine here. The intent is to use the default typography people use on the rest of the page.

I was not talking about whether this is "fine" or "ok" or any other value judgment. Again a "breaking" change does not value the change. It simply states the implications a change has.

This change might have some which is why we should add a warning to the changelog. This is not concerned with judging this change on its merits.

Reply:
Agree, I was considering the impact of the change.